datastore_mongo_test.go

```diff
@@ -634,6 +645,14 @@ func TestMongoUpsertAttributes(t *testing.T) {
 			t.Errorf("attributes mismatch, have: %v\nwant: %v", dev.Attributes, tc.outAttrs)
 		}
 
+		//check timestamp validity
+		//note that mongo stores time with lower precision- custom comparison
+		assert.Equal(t, createdTs.Unix(), dev.CreatedTs.Unix())
```

Comment:
...meh, but it won't work in Condition, will still have to keep unix times, or am I missing something